tools/accuracy_checker/accuracy_checker/utils.py

```diff
@@ -489,3 +489,11 @@ def check_file_existence(file):
         return True
     except (FileNotFoundError, IsADirectoryError):
         return False
+
+def get_stride_from_config(config, allow_none=False):
```

Comment:
As I understand, stride is required argument for your preprocessor, right?
What about set it optional=False in parameters? It will be mean that it should be always provided (None is not allowed and it will be checked automatically)
BTW why at the end you return tuple with 2 None values?